lib/nostrum/struct/message/sticker.ex

```diff
@@ -0,0 +1,122 @@
+defmodule Nostrum.Struct.Message.Sticker do
+  @moduledoc """
+  Struct representing a Discord message.
+  """
+
+  alias Nostrum.Struct.{Guild, User}
+  alias Nostrum.{Snowflake, Util}
+
+  defstruct [
+    :id,
+    :pack_id,
+    :name,
+    :description,
+    :tags,
+    :type,
+    :format_type,
+    :available,
+    :guild_id,
+    :user,
+    :sort_value
+  ]
+
+  @typedoc """
+  	id of the sticker
+  """
+  @type id :: Snowflake.t()
+
+  @typedoc """
+  for standard stickers, id of the pack the sticker is from
+  """
+  @type pack_id :: Snowflake.t()
+
+  @typedoc """
+  name of the sticker
+  """
+  @type name :: String.t() | nil
+
+  @typedoc """
+  	description of the sticker
+  """
+  @type description :: String.t() | nil
+
+  @typedoc """
+  for guild stickers, the Discord name of a unicode emoji representing the sticker's expression. for standard stickers, a comma-separated list of related expressions.
+  """
+  @type tags :: String.t() | nil
+
+  @typedoc """
+  Sticker Types:
+
+  TYPE     |	VALUE 	|DESCRIPTION
+  ---------|----------|-----------------------------------------------------------------------------
+  STANDARD |	1     	|an official sticker in a pack, part of Nitro or in a removed purchasable pack
+  GUILD    |	2     	|a sticker uploaded to a Boosted guild for the guild's members
+  """
+  @type type :: integer
```

Comment:
This docstring does not render properly in the documentation:

![image](https://user-images.githubusercontent.com/22679924/128057527-1578909c-e07b-4f3d-a186-72e5661c73e1.png)